docs/manual/upgrades.md

```diff
@@ -32,6 +32,18 @@ In the main container the new binaries will be present in `/var/dynamic-conf/bin
 The `UpdatePodConfig` subreconciler will ensure that all Pods have the new configuration present, this configuration ensures that the `fdbmonitor` will restart processes with the new binary present in the shared volume.
 Once all Pods have the new configuration and binaries present the operator can move to the next phase.
 
+A potentional blocker for moving towards the next Phase can be an issue with a subset of the Pods e.g. if the sidecar is unreachable.
+If the sidecar is unreachable the operator is not able to confirm that the new configuration and binary is in place.
+All Process Groups affected by such an issue can be discovered with the `SidecarUnreachable` condition.
+The best way forward for this is to replace the affected Pods with the following [kubectl fdb plugin](../../kubectl-fdb/Readme.md) command:
```

Comment:
The pod replacement will cause the containers/server process within the main container to get restarted, correct?

Reply:
Is there a reason we don't want to do automatic replacement (the way described below) in this case?

Reply:
> The pod replacement will cause the containers/server process within the main container to get restarted, correct?

Correct.

> Is there a reason we don't want to do automatic replacement (the way described below) in this case?

I think it should be fine to replace Pods if they have that condition. I don't recall if we made any specific decision on skipping that decision for a specific reason.

Reply:
Ok, we should do automatic replacement in this case then (if we are not doing it already). Also, we probably should make FailureDetectionTimeSeconds to be < 7200 seconds in order for the replacement to be effective. A process(es) (that has been staged correctly) can restart and can get the cluster into a bad state in 7200 seconds. Any reason we picked such a high value for FailureDetectionTimeSeconds?

Reply:
Created https://github.com/FoundationDB/fdb-kubernetes-operator/issues/1518 for that. 2 hours is the default value and can be adjusted by users if wanted. The value was picked to ensure replacements are not triggered to quickly in cases where a host is unavailable for less than 2 hours.

Reply:
Ok.